### PR TITLE
fix: [IOBP-384] Header transaction summary safe area view

### PR DESCRIPTION
**Pull request by Hantex9**
## Short description
This PR fixes the content top margin due to wrong SafeAreaView

## List of changes proposed in this pull request
- Moved `SafeAreaView` component one level upside;

## How to test
Try to pay any pagoPA notice until there is the transaction summary screen showed

## Preview
|Before|After|
|-|-|
||

### CODE REVIEW

File: ts/screens/wallet/payment/TransactionSummaryScreen.tsx [hunk 1]
```diff
@@ -471,8 +471,8 @@ const TransactionSummaryScreen = (): React.ReactElement => {
             () => startAssistanceRequest(error, messageId)
           )}
         </ContentWrapper>
-      </SafeAreaView>
-    </BaseScreenComponent>
+      </BaseScreenComponent>
```
Comment (mastro993) [inline ts/screens/wallet/payment/TransactionSummaryScreen.tsx:474]: On Android the button is too close to the bottom. 




Maybe a `VSpacer` could be enough. What do you think?
```suggestion
          <VSpacer size={16} />
      </BaseScreenComponent>
```

**Review by mastro993 — CHANGES_REQUESTED** [review]
On Android, the button is too close to the bottom. I think

File: ts/screens/wallet/payment/TransactionSummaryScreen.tsx (hunk 1 above)
Comment (Hantex9) [reply ts/screens/wallet/payment/TransactionSummaryScreen.tsx:474]: This issue is already addressed in another PR #5185

The actual PR aims to remove the heading spacing glitch.

**Review by mastro993 — APPROVED** [review]
LGTM

**Pull request merged**